plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1452,6 +1468,7 @@ elementgui.tool.usage_count=Number of uses / durability:
 elementgui.tool.blocks_affected=Blocks affected: 
 elementgui.tool.stays_in_grid_when_crafting=Does item stay in crafting grid when crafted?
 elementgui.tool.damaged_on_crafting=<html>Damage item instead on crafting<br><small>Make sure to enable "stay in crafting grid" and that item is damageable
+elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Separate entries with comma, to use comma in description use,

```

Comment:
This is not externalized in the right way. there has to be an escaped backslash and space before the comma. Please recheck the original string.

Reply:
Like food does this eg.:

`elementgui.food.tooltip_tip=<html>Special information about the food:<br><small>Separate entries with comma, to use comma in description use \\,`

Reply:
Yes I know